### PR TITLE
workflow: adding default to workflow core and is_update field

### PR DESCRIPTION
**Pull request by DonHaul**

### CODE REVIEW

File: backoffice/workflows/models.py [hunk 1]
```diff
@@ -26,8 +26,8 @@ class Workflow(models.Model):
         choices=STATUS_CHOICES,
         default=DEFAULT_STATUS_CHOICE,
     )
-    core = models.BooleanField()
-    is_update = models.BooleanField()
+    core = models.BooleanField(default=False)
+    is_update = models.BooleanField(default=False)
```
Comment (drjova) [inline backoffice/workflows/models.py:30]: we have to create also a migration for these :)

**Review by drjova — CHANGES_REQUESTED** [review]
It's missing the db migration 🙈

File: backoffice/workflows/models.py (hunk 1 above)
Comment (DonHaul) [reply backoffice/workflows/models.py:30]: added :)

**Comment by DonHaul** [comment]
@drjova it should be good now

**Review by drjova — APPROVED** [review]

**Pull request merged**